cmd/openshift-tests/provider.go

```diff
@@ -139,6 +139,19 @@ func decodeProvider(provider string, dryRun, discover bool, clusterState *exutil
 		if err := json.Unmarshal([]byte(provider), config); err != nil {
 			return nil, fmt.Errorf("provider must decode into the ClusterConfig object: %v", err)
 		}
+
+		coreClient, err := e2e.LoadClientset(true)
+		if err != nil {
+			return nil, err
+		}
+		isMicroShift, err := exutil.IsMicroShiftCluster(coreClient)
+		if err != nil {
+			return nil, err
+		}
+		if isMicroShift {
+			config.HasNoOptionalCapabilities = true
+		}
```

Comment:
are you sure you're ending on up in the "Default" provider case when running against a microshift cluster?

should be easy enough to run locally w/ some debug statements and see what's actually happening when the detection logic runs

Reply:
No, it should be inside "none" case. There was something wrong with my MicroShift cluster which leads all debug statements to go inside "Default" case. 

I've added it now in "none" case.